src/messaging/ReliableMessageProtocolConfig.cpp

```diff
@@ -135,7 +135,7 @@ System::Clock::Timeout GetRetransmissionTimeout(System::Clock::Timeout activeInt
     System::Clock::Timestamp timeout(0);
     for (uint8_t i = 0; i < CHIP_CONFIG_RMP_DEFAULT_MAX_RETRANS + 1; i++)
     {
-        auto baseInterval = ((timeSinceLastActivity + timeout) < activityThreshold) ? activeInterval : idleInterval;
```

Comment:
You shouldn't remove the usage of idle interval for SIT or non-ICD devices. This exception should be for LIT only where the idle interval can be very long.

Reply:
I agree, I think it would be to use min(activeInterval, idleInterval) here for both client and server regarding mrp configuration.

For SIT/non-ICD, SAI would be larger than SII, then it would go with SII when it reaches to idle status.

For LIT, SAI would be less than SII, then it would go with SAI when it reaches to idle status.

Reply:
I don't think SAI is usually larger than SII for SIT/non-ICD. In fact, default SAI is 300ms while SII is 500. For SIT it's also common to use a sub-second fast-polling interval and a couple of seconds slow-polling interval. Unfortunately, it seems that we need some sort of flag passed from outside to indicate that the receiver is LIT. Alternatively, we could detect that by checking if SII > 15s?

Reply:
Yes, considering dynamic ICD feature, it would be hard to feature flag to setup MRP timeout here.

I would prefer checking if SII > 15 s, I just update the code to reflect, thanks